meson.build

```diff
@@ -0,0 +1,273 @@
+#Project definition
```

Comment:
I believe we got rid of the Meson stuff. This file was probably in there when you originally started working on this PR though. It should be removed.